lldb/test/Shell/helper/stepper.py

```diff
@@ -0,0 +1,213 @@
+from __future__ import print_function
+from collections import defaultdict
+
+import os
+import re
+
+import lldb
+
+###############################################################################
+# Configurable options (see the STEPPER_OPTIONS env var)
+###############################################################################
+
+# Only visit unoptimized frames.
+ONLY_VISIT_UNOPTIMIZED = True
+
+# Only visit frames with swift code.
+ONLY_VISIT_SWIFT = False
+
+# Skip doing "frame var" on all locals, in each frame.
+SKIP_FRAMEVAR = False
+
+# Skip doing "po" on all locals, in each frame.
+SKIP_PO = False
+
+# Maximum number of times to visit a PC before "finish"ing out.
+MAX_VISITS_PER_PC = 50
+
+# Run the process. It's useful to set this to False if you have monitor/test
+# lldb's, and the child lldb can't read from stdin to do things like continue
+# at a breakpoint.
+RUN_THE_PROCESS = True
+
+# Skip N frames between each inspection.
+SKIP_N_FRAMES_BETWEEN_INSPECTIONS = 0
+
+###############################################################################
+
+
+def parse_options():
+    """
+    Parse the STEPPER_OPTIONS environment variable.
+
+    Update any constants specified in the option string.
+    """
+    global ONLY_VISIT_UNOPTIMIZED, ONLY_VISIT_SWIFT, SKIP_FRAMEVAR, \
+            SKIP_PO, MAX_VISITS_PER_PC, RUN_THE_PROCESS, \
+            SKIP_N_FRAMES_BETWEEN_INSPECTIONS
+    opts = os.getenv('STEPPER_OPTIONS', '').split(';')
+    for o in opts:
+        m = re.match('(\w+)=([^;]+)', o)
+        if not m:
+            print('Unrecognized option:', o)
+        else:
+            option, val = m.groups()
+            print('Setting', option, '=', val)
+            globals()[option] = eval(val)
+
+
+def doit(dbg, cmd):
+    "Run a driver command."
+    print('::', cmd)
+    dbg.HandleCommand(cmd)
+    return False
+
+
+def doquit(dbg):
+    "Quit the stepper script interpreter."
+    os._exit(0)
```

Comment:
If there's a reason to not call `os.exit()`, maybe a comment why?

Reply:
No reason, just `exit(0)` works fine. Will fix.